labs/a1.md

```diff
@@ -116,5 +116,38 @@ ab/abizer/some/path
 
 Hint: The easiest way to do this is with one very short command.
 
+## Question 4
+
+At some point, everyone has looked at a problem and thought to themselves:
+"Hey, I can do this in one line!"
+
+Lets find out if you can. I need to sort out some of my most listened to albums
+by making directories for each of their songs, specifically for my favorite artist Future.
+
+You need to fetch the list of albums and their respective artist from
+`https://raw.githubusercontent.com/0xcf/decal-labs/master/a1/albums.txt`.
+Filter out the albums we want, trim out the album name,
+and then make a directory for each one, all in one line.
+
+```
+$ cat albums.txt
+...
+Drip or Drown 2, Gunna
+Playboi Carti, Playboi Carti
+DS2 (Deluxe), Future
+Drip Harder, Lil Baby
+...
+
+# After our magic one liner!
+
+$ ls
+'DS2 (Deluxe)'   ...
+```
+
+Hint: Use `man` to see what arguments you need to xargs to behave the way you want.
+If you're tired of repeatedly testing commands, try using an online `sed` REPL to iterate faster.
```

Comment:
`sed` isn't strictly necessary to solve this (personally I wouldn't use it), but the way it's mentioned here gives off a strong implication that it's required. I worry that this could put the reader in a state of mind where they feel "I need to use `sed`" and work backwards to the solution, which isn't a good mindset.

I don't think it's necessary to suggest an online tool anyways. The reader can iterate and experiment in their normal command line.

I would recommend listing a few common text manipulation commands (`cut`, `sed`, `awk`, `grep`, probably others that I'm missing) which makes this more open-ended.

Reply:
Yup, this feedback is spot on. I'll tweak this at our next summer decal meeting.